src/server/service/batch.rs

```diff
@@ -35,7 +35,7 @@ pub struct ReqBatcher {
 
 impl ReqBatcher {
     pub fn new(batch_size: usize) -> ReqBatcher {
-        let begin_instant = Instant::now_coarse();
+        let begin_instant = Instant::now();
```

Comment:
may using now here affect the performance in the hot path?

Reply:
`now` takes about 20ns while `now_coarse` takes about 4ns.

As our fastest RPC takes tens of microseconds, such increase is ignorable.